assets/component-menu-drawer.css

```diff
@@ -169,24 +193,35 @@ details[open].menu-opening > .menu-drawer__submenu {
 
 .menu-drawer__menu-item > .icon-arrow {
   position: absolute;
-  right: 2.5rem;
   top: 50%;
+  right: 2.5rem;
   transform: translateY(-50%);
 }
 
+html[dir="rtl"] .menu-drawer__menu-item > .icon-arrow {
+  transform: translateY(-50%) rotate(180deg);
```

Comment:
Your comment said we shouldn't be using logical properties on elements using `transform: translate(50%)`. Is this one of them?